HRTool/src/main/java/app/controllers/ImageUploadRestController.java

```diff
@@ -14,55 +12,26 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
 
-
-
 @Controller
-
 public class ImageUploadRestController {
 
 	@Autowired
-    private ServletContext servletContext;
-
-	
-	@RequestMapping(value = "/upload", method = RequestMethod.GET)
-	public @ResponseBody String uploading(){
-		
-		
-		System.out.println("---------------------------------");
-		return "BLA BLA";
+	private ServletContext servletContext;
 
-
-	}
-	
 	@RequestMapping(value = "/upload", method = RequestMethod.POST)
-	public @ResponseBody String uploadingPOST(@RequestParam("flowFilename") String fileName,
-			@RequestParam("flowTotalChunks") int totalChunks,
-			@RequestParam("slika") MultipartFile file){
-		System.out.println("----------------POST-------------");
+	public @ResponseBody String uploadingPOST(
```

Comment:
[at]ResponseBody is not needed. Any method in the [at]RestController will automatically get it.
Do not return String. Return ResponseEntity.